kv/db_test.go

```diff
@@ -30,7 +31,12 @@ import (
 )
 
 func createTestClient(t *testing.T, addr string) *client.DB {
-	db, err := client.Open("https://root@" + addr + "?certs=" + security.EmbeddedCertsDir)
+	return createTestClientForUser(t, addr, security.NodeUser)
+}
+
+func createTestClientForUser(t *testing.T, addr, user string) *client.DB {
+	db, err := client.Open(fmt.Sprintf("https://%s@%s?certs=%s",
```

Comment:
not sure it would be too helpful, it would just end up being:
`func buildURL(scheme, user, address, path, certsDir)` without much possible simplification since not every caller has a context.